plugins/services/src/js/components/__tests__/ServiceStatusProgressBar-test.js

```diff
@@ -36,19 +37,42 @@ describe("#ServiceStatusProgressBar", function() {
       this.instance = ReactDOM.render(
         <ServiceStatusProgressBar
           service={
-            new Application({
+            new Pod({
```

Comment:
I think application exposes it, but maybe as a number instead of a list?
```
  getInstancesCount() {
    return this.get("instances") || 0;
  }
```

or maybe it does not know running?